crypto/core_algorithm.c

```diff
@@ -29,7 +29,7 @@ static int algorithm_do_this(OSSL_PROVIDER *provider, void *cbdata)
     struct algorithm_data_st *data = cbdata;
     int no_store = 0;    /* Assume caching is ok */
     int first_operation = 1;
-    int last_operation = OSSL_OP__HIGHEST;
+    int last_operation = OSSL_OP_HIGHEST;
```

Comment:
The reason for the double underscore is so that it's not mixed up with the usual operation numbers...  that is, unless you want to claim that there is a "HIGHEST" operation :wink:

Reply:
I guessed that this was your intention, but do you _really_ think such a precaution is necessary?

Reply:
Yes

Reply:
[Reverted](https://github.com/openssl/openssl/pull/12222/commits/cae2cfdff71d9315e8f42227b3a684e1d0c40d2e).